tex2pdf_service/tex2pdf/converter_driver.py

```diff
@@ -372,6 +373,10 @@ def _finalize_pdf(self) -> None:
         except AssemblingFileNotFound as exc:
             logger.warning("Failed combining PDFs: %s", exc, extra=self.log_extra)
             pass
+        except subprocess.TimeoutExpired as exc:
+            logger.warning("Failed combining PDFs: gs call timed out: %s", exc, extra=self.log_extra)
+        except subprocess.CalledProcessError as exc:
+            logger.warning("Failed combining PDFs: gs returned an error: %s", exc, extra=self.log_extra)
```

Comment:
Note that in case the `gs` run fails, the stdout and stderr are saved into the exception `exc` so will be visible.